lib/cequel/metal/row.rb

```diff
@@ -24,6 +24,7 @@ def self.from_result_row(result_row)
                 if $1 == 'ttl' then row.set_ttl($2, value)
                 else row.set_writetime($2, value)
                 end
+              elsif value.respond_to?(:in_time_zone) && value.is_a?(Time) then row[name] = value.in_time_zone
```

Comment:
Do we need the `is_a?` check here? Seems like if it responds to `in_time_zone`, that should be enough for our purposes